test/integration/pagination_test.go

```diff
@@ -19,10 +19,11 @@ import (
 	"github.com/stretchr/testify/assert"
 	authenticationv1 "k8s.io/api/authentication/v1"
 	corev1 "k8s.io/api/core/v1"
-	errs "k8s.io/apimachinery/pkg/api/errors"
 	rbacv1 "k8s.io/api/rbac/v1"
+	errs "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	sourcesv1 "knative.dev/eventing/pkg/apis/sources/v1"
```

Comment:
Shouldn't this be before unstructured?

Reply:
`sources` comes alphabetically after `meta`

Reply:
I can move it. I legitimately don't know if letters come before numbers

Reply:
I assumed (which probably makes it wrong) it was alphabetical, using the last portion of the path ("unstructured", in this case) if there was no alias.

Reply:
so `goimports` handles ordering imports in a specific order and if I move `sourcesv1` above the `k8s.io` imports, `goimports` moves `sourcesv1` back to the bottom of the import list

Reply:
Works for me.  At least now I know.